win_etw_provider/Cargo.toml

```diff
@@ -14,13 +14,16 @@ readme = "../README.md"
 widestring = "^1.0"
 zerocopy = "^0.6"
 win_etw_metadata = { version = "^0.1.2", path = "../win_etw_metadata" }
+uuid = "1"
```

Comment:
Can you make this an optional dependency / feature?

Reply:
done